### PR TITLE
fix: Fix and Improve Registration Box Design

### LINKED ISSUE #7778: Fix and Improve Registration Box Design
1. Get rid of brackets. They even show up when there is no description
Screenshot from 2021-09-13 10-39-00

2. Improve the design and make it more similar to the below screenshot.
Screenshot from 2021-08-25 15-17-25
* [ ] Make the cost bold, e.g. USD 10.00 
* [ ] Increase the spacing between the radio button and the text behind a bit
* [ ] Move the description with the same indent below the ticket name
* [ ] Make the radio button empty circle line less thick and use the same gray color as it is used elsewhere in the design

Compare current box design
Screenshot from 2021-09-13 10-51-43

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #7778 

#### Changes proposed in this pull request:

- [x] Make the cost bold, e.g. USD 10.00
- [x]  Move the description with the same indent below the ticket name
- [x]  Make the radio button empty circle line less thick and use the same gray color as it is used elsewhere in the design

#### SCREENSHOTS

Before | After 
---- | ----------
Screen Shot 2021-09-13 at 23 33 58 | Screen Shot 2021-09-13 at 23 14 22

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by mariobehling** [comment]
Please change as follows:
1. Do not change the heading "Registration"
2. The radio button should always be on the left and have the same indent with all registration options
Screenshot from 2021-09-14 13-58-30
3. The radio box is no longer visible. The line is too thin or the color is too bright.

**Comment by mariobehling** [comment]
Thanks for taking care of the color change as well. Please change that to light gray same as the buttons, e.g. on the right hand-side. This seems to be more aligned with the color of the theme. 

Screenshot from 2021-09-14 14-02-18

**Comment by daretobedifferent18** [comment]
Desktop | Mobile
------- | ------
Screen Shot 2021-09-15 at 13 33 47 | Screen Shot 2021-09-15 at 13 33 19

File: app/styles/pages/events.scss [hunk 1]
```diff
@@ -14,6 +14,33 @@
   }
 }
 
+.select-ticket {
+  background-color: #e0e1e2 !important;
+}
+
+.custom-checkbox {
```
Comment (maze-runnar) [inline app/styles/pages/events.scss:21]: rename it to something like `.custom-registration-checkbox`

File: app/styles/pages/events.scss [hunk 2]
```diff
@@ -14,6 +14,33 @@
...
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 22px;
+}
+
+.custom-bullet {
```
Comment (maze-runnar) [inline app/styles/pages/events.scss:30]: here too.

**Comment by daretobedifferent18** [comment]
Done @maze-runnar

**Review by maze-runnar — DISMISSED** [review]

**Review by mariobehling — APPROVED** [review]
Thank you

**Comment by maze-runnar** [comment]
@daretobedifferent18 update branch

**Comment by mariobehling** [comment]
Please resolve merge conflicts. Thanks.

File: app/templates/public/index.hbs [hunk 3]
```diff
@@ -8,49 +8,56 @@
...
+                  <div class="twelve wide column">
+                    <div class="row">{{ticket.name}}</div>
+                    <div class="row">
+                      <small class="ui gray-text tiny">{{if ticket.description ticket.description}}</small>
+                      <br>
+                      {{#if (and this.model.tax (not-eq ticket.type 'freeRegistration'))}}
+                        <p>
+                          {{#if this.model.tax.isTaxIncludedInPrice}}
+                            <small class="ui gray-text small">
+                              {{t 'includes'}} <CurrencyAmount @currency={{this.model.event.paymentCurrency}} @amount={{ticket.includedTaxAmount}}/>
```
Comment (mariobehling) [inline app/templates/public/index.hbs:32]: Looks like you overwrote the changes of the previous PR. Please change this back. Compare https://github.com/fossasia/open-event-frontend/pull/7804/files

Comment (daretobedifferent18) [reply app/templates/public/index.hbs:32]: It must have came from merge conflict maybe. Let me fix this

Comment (daretobedifferent18) [reply app/templates/public/index.hbs:32]: @mariobehling  its not here

File: app/templates/public/index.hbs [hunk 4]
```diff
@@ -8,49 +8,56 @@
...
+                            </small>
+                          {{else}}
+                            <small class="ui gray-text small">
+                              + <CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
```
Comment (daretobedifferent18) [inline app/templates/public/index.hbs:36]: Let me do this by creating new PR.

Comment (daretobedifferent18) [reply app/templates/public/index.hbs:36]: #7808

**Pull request merged**